### DISCUSSION TITLE
Onion Skin (editor)

### DISCUSSION THREAD

**Discussion by PhoenixAndStuf** [ScratchAddons/ScratchAddons#866] [Ideas] [answered]
Posted this in the feedback section on your website, but just wanted to get word out about my idea:

Onion skinning. Onion skinning is a tool often used by animators to see the previous and the next frame from the frame they are drawing in. Useful stuff to make more precise animations. 

Animation, as a whole, is useful for game makers too. Who wants to see a box moving around a screen? I would prefer an animated character and I garuentee you would too. But anyway, back to the point.

Onion skinning has been missing from scratch since scratch has existed. Im thinking the addon could be a toggle on / off button near where all of the other tools are.

Disclaimer: I am not a coder and I have no idea how hard coding this could be, so take this suggestion with a grain of salt.

**Comment by TheColaber** [comment, marked as answer]
Duplicate of #193

**Reply by PhoenixAndStuf** [reply]
Oh, ok

**Reply by WorldLanguages** [reply]
@TheColaber Bruh be a bit nicer to newcomers :p

**Reply by TheColaber** [reply]
Sorry...

**Reply by Explosion-Scratch** [reply]
XD

**Reply by Explosion-Scratch** [reply]
@PhoenixAndStuf Thanks for checking this out though! Feel free to suggest any new ideas you have for ScratchAddons!

**Reply by PhoenixAndStuf** [reply]
> @TheColaber Bruh be a bit nicer to newcomers :p

I've been around the scratch forums though, so I'm a little used to it.

**Reply by PhoenixAndStuf** [reply]
> @PhoenixAndStuf Thanks for checking this out though! Feel free to suggest any new ideas you have for ScratchAddons!

Alright, thanks :)
